doc/python/static-image-export.md

```diff
@@ -64,6 +64,19 @@ If you are unable to install conda or npm, you can install orca as a precompiled
 ```
 $ pip install psutil requests
 ```
+
+##### Externally Hosted Orca + pip
```

Comment:
why "+ pip" ?

Reply:
> why "+ pip" ?

I copied the title style of the other subsections. Would you prefer me to remove it?

P.S. One still needs to install dependencies via `pip` namely `requests` and `psutil` (the requirement for the latter could/should be removed but atm it's mandatory) for this feature to work.